vendor/src/github.com/docker/notary/client/delegations.go

```diff
@@ -0,0 +1,294 @@
+package client
+
+import (
+	"encoding/json"
```

Comment:
@tiborvass we have an issue to track switching all of notary to canonical json https://github.com/docker/notary/issues/481.

The current notary client already uses the same `encoding/json` package and we'll need additional refactoring throughout all of notary to make the full switch to canonical json.  cc @endophage @cyli